packages/moonstone/Scrollable/ScrollableNative.js

```diff
@@ -560,6 +570,60 @@ class ScrollableBaseNative extends Component {
 		}
 	}
 
+	updateFocusAfterVoiceControl = () => {
+		const
+			{direction} = this.props,
+			viewportBounds = this.uiRef.containerRef.getBoundingClientRect(),
+			spotItemBounds = document.activeElement.getBoundingClientRect(),
+			nodes = this.uiRef.containerRef.querySelectorAll('[data-index]'),
+			isVertical = (direction === 'vertical' || direction === 'both'),
+			first = isVertical ? 'top' : 'left',
+			last = isVertical ? 'bottom' : 'right';
+
+		if (spotItemBounds[last] < viewportBounds[first] || spotItemBounds[first] > viewportBounds[last]) {
+			for (let i = 0; i < nodes.length; i++) {
+				const nodeBounds = nodes[i].getBoundingClientRect();
+				if (nodeBounds[first] > viewportBounds[first] && nodeBounds[last] < viewportBounds[last]) {
```

Comment:
I don't think that it is not the good solution to find a focusable node because nodes[i+1] could be positioned upper than nodes[i] with absolute or relative position.

Reply:
If there is a better solution, I will improve it.